datalab/settings.py

```diff
@@ -37,6 +37,7 @@ def get_list_from_env(variable, default=None):
 
 # Build paths inside the project like this: BASE_DIR / 'subdir'.
 BASE_DIR = Path(__file__).resolve().parent.parent
+TEMP_FITS_DIR = os.getenv('TMP_DIR', BASE_DIR / 'tmp' / 'fits')
```

Comment:
Just for clarity we should probably keep the environment variable the same as the variable name in the settings module, so 

```
TEMP_FITS_DIR = os.getenv('TEMP_FITS_DIR', BASE_DIR / 'tmp' / 'fits')
```

And I don't think the default you've supplied will actually work - it needs to be a string path and Python will try and use the `/` operator as a division operator by default

Reply:
import os

```
TEMP_FITS_DIR = os.getenv('TMP_DIR', os.path.join(BASE_DIR, 'tmp/fits/')
```